api/main_endpoints/routes/User.js

```diff
@@ -509,4 +509,39 @@ router.post('/usersSubscribedAndVerified', function(req, res) {
     });
 });
 
+//For Club Revenue page 
+router.post('/countMembers', async (req, res) => {
+  if (!checkIfTokenSent(req)) {
+    return res.sendStatus(FORBIDDEN);
+  } else if (!checkIfTokenValid(req)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+  
+  const currentYear = new Date().getFullYear();
+
+  const jan1ThisYear = new Date(currentYear, 0, 1);
+  const june1ThisYear = new Date(currentYear, 5, 1);
+  const jan1NextYear = new Date(currentYear + 1, 0, 1);
+  const june1NextYear = new Date(currentYear + 1, 5, 1);
+  const dec31ThisYear = new Date(currentYear, 11, 31);
+
+  const today = new Date();
+
+  let count, newSingleSemester, newAnnualMembers, totalNewMembersThisYear, currentActiveMembers;
+
+  totalNewMembersThisYear = await User.countDocuments({ emailVerified: true, accessLevel: 1, joinDate: { $gte: jan1ThisYear, $lte: today } });
+  currentActiveMembers = await User.countDocuments({ emailVerified: true, accessLevel: 1, });
```

Comment:
can you use the constants.MEMBERSHIP_STATE in the below file instead of the integer value

https://github.com/SCE-Development/Clark/blob/e2a203448f783ffdbdf581c8b8fd5dd4c6ec5f63/api/util/constants.js#L11